app/javascript/components/visualization/ScatterPlot.js

```diff
@@ -233,6 +233,12 @@ function getLegendEntries(data, pointSize, labelCorrelations) {
   return legendEntries
 }
 
+
+  // Reverse the continous colorscale as appropriate for high contrast color to correspond to high expression
+  function shouldReverseScale(scatterColor) {
+    return scatterColor === 'Reds' ? false : true
+  }
```

Comment:
More succinctly, and with formatting that doesn't throw lint warnings:

```suggestion
/** Reverse the continuous colorscale so high contrast color corresponds to high expression */
function shouldReverseScale(scatterColor) {
  return scatterColor !== 'Reds'
}
```